src/Microsoft.DotNet.Build.Tasks/PackageFiles/resources.targets

```diff
@@ -46,14 +46,18 @@
     </EmbeddedResource>
   </ItemGroup>
   
-  <ItemGroup Condition="Exists('$(StringResourcesPath)') And '$(SkipCommonResourcesIncludes)'=='' AND '$(OmitResources)'!='true'">
-    <Compile Condition="'$(MSBuildProjectExtension)' == '.csproj'" Include="$(CommonPath)/System/SR.cs">
-      <Visible>true</Visible>
-      <Link>Resources/Common/SR.cs</Link>
-    </Compile>
-    <Compile Condition="'$(MSBuildProjectExtension)' == '.vbproj'" Include="$(CommonPath)/System/SR.vb">
-      <Visible>true</Visible>
-      <Link>Resources/Common/SR.vb</Link>
-    </Compile>
-  </ItemGroup>
+  <Choose>
+    <When Condition="Exists('$(StringResourcesPath)') And '$(SkipCommonResourcesIncludes)'=='' AND '$(OmitResources)'!='true'">
+      <ItemGroup>
+        <Compile Condition="'$(MSBuildProjectExtension)' == '.csproj'" Include="$(CommonPath)/System/SR.cs">
+          <Visible>true</Visible>
```

Comment:
Link is just metadata that tell VS what folder to put this in when displaying in solution explorer. So if it isn't being displayed any longer then it doesn't really add any value.